### PR TITLE
Issue 16 - Utility 3 - Add tokens in js

### PR DESCRIPTION
**Pull request by FrancescoSTL**
This PR is a branch of https://github.com/mozilla/all-aboard/pull/26 which adds tokens to the sidebar in `content3.js`.

Issue ref: #16

### CODE REVIEW

**Comment by schalkneethling** [comment]
I will wait until https://github.com/mozilla/all-aboard/pull/26 has been merged and this pull request rebased before reviewing this one.

**Comment by schalkneethling** [comment]
Needs a rebase

**Comment by FrancescoSTL** [comment]
Donezo 👍

File: data/js/utility/content3.js [hunk 1]
```diff
@@ -4,8 +4,16 @@
...
+    addon.port.on('tokens', function(tokens) {
+        for (var i = 0, l = tokens.length; i < l; i++) {
+            // set the token container to active and shows the token
+            document.querySelector('#' + tokens[i]).classList.add('active');
+        }
+    });
+
     // add listeners to all of our buttons for clicking
-    for(var x=0, l=templateNames.length; x<l; x++)
+    for(var x=0; x<templateNames.length; x++)
```
Comment (schalkneethling) [inline data/js/utility/content3.js]: Did you mean to change this back?

File: data/js/utility/content3.js [hunk 2]
```diff
@@ -24,4 +32,4 @@
         // notify addon that we've clicked the button
         addon.port.emit('intent', 'defaultTemplate');
     });
-})();
+})();
```
Comment (schalkneethling) [inline data/js/utility/content3.js]: Whoops, missing empty line at the end of the file.

File: index.js [hunk 3]
```diff
@@ -512,4 +512,4 @@ function init() {
     }
 }
 
-init();
+init();
```
Comment (schalkneethling) [inline index.js]: Here as well

**Comment by schalkneethling** [comment]
r+ 🗼

**Pull request merged**